plugins/transforms/tokenreplacement/src/main/doc/tokenreplacement.adoc

```diff
@@ -0,0 +1,19 @@
+:documentationPath: /plugins/transforms/
+:language: en_US
+:page-alternativeEditUrl: https://github.com/apache/incubator-hop/edit/master/plugins/transforms/abort/src/main/doc/abort.adoc
```

Comment:
change the url to match the filename of this file

Reply:
I forget to cleanup this file. I agree to do this.